partner_financial_risk/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+======================
+Partner Financial Risk
+======================
+
+Adds a new page in partner to manage its *Financial Risk*.
+
+If any limit is exceed the partner gets forbidden to confirm customer invoice.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to *Invoicing > Configuration > Settings > Invoicing & Payments*
+#. Go to *Financial Risk* and set *Due Margin* (This field adds days of margin
+   before that a invoice is consider as unpaid)
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to *Customers > Financial Risk*
+#. Set limits and choose options to compute in credit limit.
+#. Go to *Invoicing > Sales > Customer invoices* and create new customer
```

Comment:
`#. Test the restriction trying to create an invoice for the partner for an amount higher of the limit you have set.`